src/components/PeoplePage/PeoplePage.jsx

```diff
@@ -0,0 +1,23 @@
+import React, { useEffect, useState } from 'react';
+
+import { getPeople } from '../../api/api';
+
+import { PeopleTable } from '../PeopleTable/PeopleTable';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState([]);
+
+  useEffect(() => {
+    getPeople()
+      .then(setPeople);
+  }, []);
```

Comment:
You can fetch people right directly in your PeopleTable component, not to pass them by props. Actually, you really don't need the `people` array inside PeoplePage component